tests/cloner_test.go

```diff
@@ -2378,15 +2300,22 @@ var _ = Describe("all clone tests", func() {
 			dataVolume, err := utils.CreateDataVolumeFromDefinition(f.CdiClient, targetNs.Name, targetDV)
 			Expect(err).ToNot(HaveOccurred())
 
+			targetPvc, err := utils.WaitForPVC(f.K8sClient, dataVolume.Namespace, dataVolume.Name)
+			Expect(err).ToNot(HaveOccurred())
+
+			if targetPvc.Spec.DataSourceRef != nil {
+				// Skipping with csi because force bind early causes to succeed very quickly
+				// cannot catch pod
+				Skip("only for non csi-clone")
+			}
+
+			f.ForceBindIfWaitForFirstConsumer(targetPvc)
+
 			cloneType := utils.GetCloneType(f.CdiClient, dataVolume)
 			if cloneType != "copy" {
 				Skip("only valid for copy clone")
```

Comment:
wasn't this skip enough?

Reply:
With populators don't know clone type until populator starts working (first consumer triggered)